relay-general/src/store/normalize/spans.rs

```diff
@@ -0,0 +1,727 @@
+use std::collections::BTreeSet;
+use std::collections::HashMap;
+
+use serde::{Deserialize, Serialize};
+
+use crate::protocol::{Event, Span, Timestamp};
+use crate::types::Annotated;
+
+#[derive(Clone, Copy, Debug, Eq, PartialEq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
+pub enum SpanAttribute {
+    #[serde(rename = "exclusive-time")]
+    ExclusiveTime,
+
+    /// forward compatibility
+    #[serde(other)]
+    Unknown,
+}
+
+#[derive(Clone, Debug)]
+struct TimeInterval {
```

Comment:
This looks identical to `super::breakdowns::TimeWindowSpan`. Can we unify these?

Reply:
Is there a good place to put the unified type? Perhaps in `relay-general/src/protocol/types.rs`?

Reply:
@Zylphrex just export `TimeWindowSpan` with `pub` and `use` them in `spans.rs`.

Reply:
Suggestion from @dashed sounds good until we have more use for it. Alternatively, you could go for introducing a new module `normalize/utils` that houses such types, but that is a bit of an overkill at this point. 

Putting it in `protocol` doesn't feel right to me as this is not part of the Event protocol (schema).